### DISCUSSION TITLE
Running model without training

### DISCUSSION THREAD

**Discussion by draxx318** [TheLastBen/fast-stable-diffusion#2274] [Q&A] [answered]
Hello, im wondering if its possible to run a model without training on my images?

**Comment by TheLastBen** [comment]
yes, use the A1111 notebook

**Reply by draxx318** [reply]
I am using it, but if i run the train step without adding images i get an error.

**Reply by draxx318** [reply]
Do i just skip the the training step and go directly to Test the trained model? Im not sure if i tried that.
I am using the notebook from the step by step guide.
https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/fast-DreamBooth.ipynb

**Reply by TheLastBen** [reply, marked as answer]
this is the A1111 notebook https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/fast_stable_diffusion_AUTOMATIC1111.ipynb (if you have colab pro)

**Reply by draxx318** [reply]
Thank you!!!
